### ISSUE TITLE
Bug: Minify gives same name to multiple different variables.

### ISSUE THREAD

**Issue by ybouane** [putoutjs/minify#6]
This code:

```js
import { minify } from '@putout/minify';

console.log(
  minify(`
    let aaaaa = 5;
    fn((a)=>{
        console.log(aaaaa);
        console.log(a);
    });
`)
);

```

Outputs:

```js
let a=5;fn(a=>{console.log(a);console.log(a)});
```

aaaaa is renamed into a but in the callback function, the argument a isn't renamed into something different.

**Comment by coderaiser** [comment]
Just fixed in [`@putout/minify@2.9.0`](https://github.com/putoutjs/minify/releases/tag/v2.9.0) 🎉. Is it working for you now?

**Comment by ybouane** [comment]
There still seems to be issues, here's another example:
```js
import { minify } from "@putout/minify";

console.log(
  minify(`
    let aaa = 5;
    fn((a, b)=>{
        console.log(b, aaa);
    });
`),
);

```

it minifies it to:
`let a=5;fn((a,b)=>console.log(b,a));`

**Comment by coderaiser** [comment]
Just fixed in [@putout/minify@2.10.0](https://github.com/putoutjs/minify/releases/tag/v2.10.0) 🎉. Is it working for you now?

**Comment by ybouane** [comment]
Yes, it seems to be working!
Here's another case that can be further improved:
```js
import { minify } from "@putout/minify";

console.log(
  minify(`
    let aaa = 5;
    fn(()=>{
        console.log(aaa, a);
    });
`),
);
```
outputs:
`let a=5;fn(()=>console.log(a,a));`

**Comment by coderaiser** [comment]
Just fixed in [@putout/minify@2.11.0](https://github.com/putoutjs/minify/releases/tag/v2.11.0) 🎉. Is it working for you now?

**Comment by ybouane** [comment]
Perfect!
